Makefile

```diff
@@ -5,6 +5,7 @@ GITSHA:=$(shell git rev-parse HEAD)
 # Get the current local branch name from git (if we can, this may be blank)
 GITBRANCH:=$(shell git symbolic-ref --short HEAD 2>/dev/null)
 GOFMT_FILES?=$$(find . -not -path "./vendor/*" -name "*.go")
```

Comment:
find . -path ./vendor -prune -o -name "*.go"
is "more correct".

Reply:
doh! 
That's very odd behavior from gofmt isn't it? I can't think of a text processing command (grep, sed, awk, etc)  that when $1 is a directory takes it upon itself to process all files inside let alone recurse! Hmm, gofmt can't even handle combined flags?!

Reply:
yeah gofmt is a bit of an odd duck... Most go tools now let you do something like `./...` to mean every go file not in the vendor directory, but they don't want to support that with gofmt on the bet that vendor directories will disappear. https://github.com/golang/go/issues/22173

> Without an explicit path, it processes the standard input. Given a file, it operates on that file; given a directory, it operates on all .go files in that directory, recursively. (Files starting with a period are ignored.) By default, gofmt prints the reformatted sources to standard output.

https://golang.org/cmd/gofmt/